src/jbi/bugzilla.py

```diff
@@ -227,6 +227,15 @@ def map_as_jira_comment(self):
         comment: BugzillaWebhookComment = self.bug.comment
         commenter: BugzillaWebhookUser = self.event.user
         comment_body: str = comment.body
+
+        if comment.is_private:
+            bug_comments = get_bugzilla().get_comments([self.bug.id])
```

Comment:
I don't think we should call `get_bugzilla()`  from here. It does not feel like it's the right way.

I would try something like that:
```py
if comment.is_private:
     all_comments = self.bugzilla_object.fetch_all_comments()
     matching_comments = [c for c in all_comments if c["id"] == comment.id]
     ...
```


I now realize we need to rename these  *map* methods, because the current state is confusing, we have `map_as_jira_comment` and `map_as_comment`. And while reading your PR it took me some time to understand them. One seems to be `bugzilla_comment_to_jira_comment`  and the other one `bugzilla_change_as_jira_comment` (not saying these are great names).

Reply:
> so I'm not sure what the improvement is there

In terms of readability, it would be a little improvement, but in terms of performance, you are totally right actually. The reason I was nervous about this is #140 and we can tackle that in another PR.